matlab/delta.m

```diff
@@ -0,0 +1,18 @@
+
+function [dLat dLng] = delta(lat,lng)
+    
+    earthRadius = 6378245 % Replace from because of differnet ellipsoid 6378137.0;
+    ee = 0.00669342162296594323;
+    
+    [dLat, dLng] = transform(lat, lng);
+    
+    radConv = pi/180;
+    radLat = lat * radConv;
+    magic = 1 - ee * sin(radLat).^2;
+    sqrtMagic = sqrt(magic);
+
+    dLat = (dLat ./((earthRadius * (1 - ee))./(magic.*sqrtMagic))./radConv);
```

Comment:
Just a side note here: see how this thing calculates deviation in meters and then converts it into degrees? And see how `transform` starts with a hardcoded 100m/300m drift? This whole thing is literally designed to fuck you up.

Reply:
Yes and all the harmonics in the shift..  
I started my work on this cause I had points on a real map that I couldn’t get to geo rectifying and affine transforms where use less and then went down this rabbit hole.  
What’s worse is that at Very small scale it looks ok and a very large scale it’s ok it just intermediate scales that suck so fining accuract áreas and distances between points more than 2 km apart really sucks